docs/source/community/code_contributions.rst

```diff
@@ -91,9 +91,7 @@ You can run locally ``scripts/cppcheck.sh`` that is a wrapper script around the
 cppcheck utility. It is known to work with cppcheck 1.61 of Ubuntu Trusty 14.0,
 since this is what is currently used on Travis-CI
 (``travis/linux_gcc/before_install.sh``).
-At the time of writing, this also works with cppcheck 1.72 of Ubuntu Xenial
-16.04, and latest cppcheck
-master.
+At the time of writing, this works with the latest cppcheck master.
```

Comment:
In `scripts/cppcheck.sh` you mention that we are using version 1.61 - which is true?

Reply:
hmm, perhaps this last phrase should be taken out, as I certainly saw some new errors with cppcheck 1.89.

Reply:
Or possibly just refrased to say that we use cppcheck as part of the quality control of the code. It is futile to try to remember to update this every time a new version of the software is used.